Source/Particles/PhysicalParticleContainer.cpp

```diff
@@ -1202,7 +1202,7 @@ PhysicalParticleContainer::AddPlasma (PlasmaInjector const& plasma_injector, int
         const bool rz_random_theta = m_rz_random_theta;
 #endif
         amrex::ParallelForRNG(overlap_box,
-        [=] AMREX_GPU_DEVICE (int i, int j, int k, amrex::RandomEngine const& engine) noexcept
+        [=] AMREX_GPU_DEVICE (int i, int j, int k, amrex::RandomEngine const& engine) noexcept //NOLINT(clang-analyzer-optin.performance.Padding)
```

Comment:
Does this belong in AMReX instead?

Reply:
I think that it is something that can happen with any lambda function (e.g., you can have a look here: https://godbolt.org/z/fK6PhvdsG). I suppose it depends on the order in which the variables captured by the lambda function are actually used.

Reply:
Let's open this as an issue in AMReX, could be interesting for @WeiqunZhang.